server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/ToSenderDomainRepository.java

```diff
@@ -37,21 +43,29 @@
  *  For example for the value 'cassandra://var/mail/sendersRepositories/', a mail sent by 'user@james.org'
  *  will be stored in 'cassandra://var/mail/sendersRepositories/james.org'.
  *  - "passThrough" optional, defaults to false. If true, the processing of the mail continues. If false it stops.
+ *  - "allowRepositoryCreation" optional, defaults to true. If true, non existing repository will be created. In case of
+ *  misconfiguration, this might lead to arbitrary repository creation. If false, the incoming mails will be stored only
+ *  in already existing repository. If not existing, the email will be dropped with an appropriate log warning (leading
+ *  to potential data loss).
```

Comment:
Could we indicate to the admin how to create a mail repository he might need, and have a link to webadmin documentation?